applications/HDF5Application/custom_python/add_custom_utilities_to_python.cpp

```diff
@@ -81,7 +81,9 @@ void AddCustomUtilitiesToPython(pybind11::module& rModule)
 
     pybind11::class_<HDF5::Detail::VertexContainerType, HDF5::Detail::VertexContainerType::Pointer>(rModule, "VertexContainer")
         .def(pybind11::init<>())
-        .def("push_back", &HDF5::Detail::VertexContainerType::push_back)
+        .def("push_back", static_cast<void (HDF5::Detail::VertexContainerType::*)
+            (HDF5::Detail::VertexContainerType::const_pointer&)>
+            (&HDF5::Detail::VertexContainerType::push_back))
```

Comment:
```suggestion
        .def("push_back", pybind11::overload_cast<const HDF5::Detail::VertexContainerType::pointer&>(&HDF5::Detail::VertexContainerType::push_back))
```

we're on C++17, so I think this should also work but admittedly, it's not much nicer in this case so I'll leave it up to you